oauthproxy.go

```diff
@@ -408,12 +423,12 @@ func buildSignInMessage(opts *options.Options) string {
 	return msg
 }
 
-func buildProviderName(p providers.Provider, override string) string {
-	if override != "" {
-		return override
-	}
-	return p.Data().ProviderName
-}
+// func buildProviderName(p providers.Provider, override string) string {
```

Comment:
to be reworked once it is decided on whether to build a custom sign-in prompt for each provider or not